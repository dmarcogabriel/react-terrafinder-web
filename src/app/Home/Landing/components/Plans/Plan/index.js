import React from 'react';
import cn from 'classnames';
import { MdClose, MdCheck } from 'react-icons/md';
import {
  container,
  header,
  title,
  plan as planStyle,
  content,
  buttonStyle,
  planKind,
  featureStyle,
  featureName,
  iconStyle,
} from './Plan.module.scss';

export default function Plan({ plan, i }) {
  return (
    <div className={container} style={{ zIndex: i === 1 ? 500 : 0 }}>
      <div className={header}>
        <p className={cn(`text-${plan.color}`, title)}>{plan.amount}</p>

        <p className={planStyle}>{plan.plan}</p>
      </div>

      <div className={cn(`bg-${plan.bgColor}`, content)}>
        <p className={planKind}>{plan.kind}</p>

        <ul>
          {plan.features.map((feature) => (
            <li key={feature.id} className={featureStyle}>
              {feature.included ? (
                <MdCheck className={iconStyle} />
              ) : (
                <MdClose className={iconStyle} />
              )}
              <span className={featureName}>{feature.name}</span>
            </li>
          ))}
        </ul>

        <button className={cn(`text-${plan.color}`, buttonStyle)} type="button">
          Quero esse
        </button>
      </div>
    </div>
  );
}
