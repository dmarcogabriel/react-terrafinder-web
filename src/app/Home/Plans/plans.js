export default [
  {
    id: 'p1',
    amount: 'Grátis',
    kind: 'Básico',
    color: 'lime-green',
    bgColor: 'lime-green',
    features: [
      {
        id: 'f1',
        name: 'Duração de 7 dias',
        included: true,
      },
      {
        id: 'f2',
        name: 'Até 15 fotos',
        included: true,
      },
      {
        id: 'f3',
        name: 'Sem demarcação do imóvel no mapa',
        included: false,
      },
      {
        id: 'f4',
        name: 'Sem suporte',
        included: false,
      },
    ],
  },
  {
    id: 'p2',
    amount: 'R$ 50',
    plan: 'por anúncio',
    kind: 'PREMIUM',
    color: 'yellow',
    bgColor: 'yellow',
    features: [
      {
        id: 'f1',
        name: 'Duração do anúncio de 90 dias',
        included: true,
      },
      {
        id: 'f2',
        name: 'Visibilidade aumentada na plataforma',
        included: true,
      },
      {
        id: 'f3',
        name: 'Até 30 fotos e 1 vídeo',
        included: true,
      },
      {
        id: 'f4',
        name: 'Sem filmagens por drone nem tratamento de fotos',
        included: false,
      },
    ],
  },
  {
    id: 'p3',
    amount: 'A definir',
    plan: 'Faça uma cotação com nossos especialistas',
    kind: 'Profissional',
    color: 'pink',
    bgColor: 'blue',
    features: [
      {
        id: 'f1',
        name: 'Duração de até 365 dias',
        included: true,
      },
      {
        id: 'f2',
        name: 'Maior visibilidade da plataforma',
        included: true,
      },
      {
        id: 'f3',
        name: 'Gravação da propriedade com drones e tratamento de imagens',
        included: false,
      },
      {
        id: 'f4',
        name: 'Suporte exclusivo',
        included: false,
      },
    ],
  },
];
