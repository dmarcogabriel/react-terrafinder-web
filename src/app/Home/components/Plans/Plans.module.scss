@import 'styles';

.container {
  .title {
    font-weight: normal;
    font-size: 2rem;
    margin: 2rem 0;
    text-align: center;
  }

  .subtitle {
    font-weight: normal;
    font-size: 1.25rem;
    margin: 2rem 0;
    text-align: center;
  }

  .plans {
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;

    @media ($desktop) {
      flex-direction: row;
    }
  }

  .plansDescription {
    padding: 0 2rem;
    margin: 1rem 0 2rem;

    @media ($desktop) {
      padding: 0 10rem;
    }

    > p {
      margin: 1rem 0;
      color: $gray-dark;
      line-height: 2rem;
    }
  }
}
