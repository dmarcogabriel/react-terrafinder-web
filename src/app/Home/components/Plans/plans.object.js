import { purple } from '@mui/material/colors';

export const PLANS = [
  {
    id: 'p1',
    amount: 'Grátis',
    kind: 'Básico',
    color: '#32cd8b',
    bgColor: '#32cd8b',
    link: '/',
    features: [
      {
        id: 'f1',
        name: 'Duração de 7 dias',
        included: true,
      },
      {
        id: 'f2',
        name: 'Até 15 fotos',
        included: true,
      },
      {
        id: 'f3',
        name: 'Sem demarcação do imóvel no mapa',
        included: false,
      },
      {
        id: 'f4',
        name: 'Sem suporte',
        included: false,
      },
    ],
  },
  {
    id: 'p2',
    amount: 'R$ 129,00',
    plan: 'por anúncio',
    kind: 'PREMIUM',
    color: '#cd8632',
    bgColor: '#cd8632',
    link: '/',
    features: [
      {
        id: 'f1',
        name: 'Duração do anúncio de 90 dias',
        included: true,
      },
      {
        id: 'f2',
        name: 'Visibilidade aumentada na plataforma',
        included: true,
      },
      {
        id: 'f3',
        name: 'Até 30 fotos e 1 vídeo',
        included: true,
      },
      {
        id: 'f4',
        name: 'Sem filmagens por drone nem tratamento de fotos',
        included: false,
      },
    ],
  },
  {
    id: 'p3',
    amount: 'A definir',
    plan: 'Faça uma cotação com nossos especialistas',
    kind: 'Profissional',
    color: purple[300],
    bgColor: ' #3279cd',
    link: '/contact-us',
    features: [
      {
        id: 'f1',
        name: 'Duração de até 365 dias',
        included: true,
      },
      {
        id: 'f2',
        name: 'Maior visibilidade da plataforma',
        included: true,
      },
      {
        id: 'f3',
        name: 'Gravação da propriedade com drones e tratamento de imagens',
        included: true,
      },
      {
        id: 'f4',
        name: 'Suporte exclusivo',
        included: true,
      },
    ],
  },
];
