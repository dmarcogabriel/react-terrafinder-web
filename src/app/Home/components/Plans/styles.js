import React from 'react';
import { styled } from '@mui/material/styles';
import { Typography, Box } from '@mui/material';

export const PlansTitle = styled((props) => (
  <Typography variant="h4" {...props} />
))({
  textAlign: 'center',
  fontWeight: 'bolder',
});

export const PlansSubtitle = styled((props) => (
  <Typography variant="h5" {...props} />
))({
  textAlign: 'center',
});

export const PlansContent = styled(Box)(({ theme }) => ({
  display: 'flex',
  flexDirection: 'column',
  justifyContent: 'center',
  alignItems: 'center',
  [theme.breakpoints.up('md')]: {
    flexDirection: 'row',
  },
}));

export const PlansDescriptionText = styled(Typography)({
  color: '#4b4b4b',
  lineHeight: '2rem',
});
