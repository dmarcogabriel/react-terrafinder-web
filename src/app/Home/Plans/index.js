import React from 'react';
import classes from './Plans.module.scss';
import Plan from './Plan';
import PLANS from './plans';

export default function Plans() {
  return (
    <div className={classes.container}>
      <h1 className={classes.title}>Conheça nosso planos de anúncio</h1>
      <h2 className={classes.subtitle}>
        Escolha o que melhor se encaixa para você
      </h2>

      <div className={classes.plans}>
        {PLANS.map((plan, i) => (
          <Plan key={plan.id} plan={plan} i={i} />
        ))}
      </div>
    </div>
  );
}
