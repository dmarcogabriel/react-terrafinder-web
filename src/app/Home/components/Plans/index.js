import React from 'react';
import classes from './Plans.module.scss';
import Plan from './Plan';
import PLANS from './plans';

export default function Plans() {
  return (
    <div className={classes.container}>
      <h1 className={classes.title}>Conheça nosso planos de anúncio</h1>
      <h2 className={classes.subtitle}>
        Escolha o que melhor se encaixa para você
      </h2>

      <div className={classes.plans}>
        {PLANS.map((plan, i) => (
          <Plan key={plan.id} plan={plan} i={i} />
        ))}
      </div>

      <div className={classes.plansDescription}>
        <p>
          <b>Gratuito:</b> Recomendado para iniciantes, é uma ótima forma de
          testar a plataforma. Você pode testar a plataforma. Você pode criar um
          anúncio gratuíto da sua propriedade, com as informações gerais do
          imóvel, e dura 7 dias.
        </p>

        <p>
          <b>Premium:</b> Recomendado para quem já tem conteúdos da propriedade
          como fotos, detalhes do imóvel como tipo de solo, atividades,
          topografia. Ele custa R$ 50,00 e seu anúncio ganha mais visibilidade
          dentro da plataforma.
        </p>

        <p>
          <b>Proffissional:</b> Entre em contato conosco para conversarmos e
          conhecermos melhor o seu imóvel rural. Iremos até a propriedade,
          colheremos as informações mais importanjtes dela e faremos excelentes
          gravações explorando o imóvel através de dones de ponta. Essa é a
          maneira mais fácil de vender.
        </p>
      </div>
    </div>
  );
}
