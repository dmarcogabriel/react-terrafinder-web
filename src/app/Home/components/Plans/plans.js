import React from 'react';
import { Box } from '@mui/material';
import Plan from './Plan';
import { PLANS } from './plans.object';
import {
  PlansTitle,
  PlansSubtitle,
  PlansContent,
  PlansDescriptionText,
} from './styles';

export const Plans = () => (
  <Box sx={{ py: 2, px: 2 }}>
    <PlansTitle sx={{ my: 2 }}>Conheça nossos planos de anúncio</PlansTitle>
    <PlansSubtitle sx={{ my: 2 }}>
      Escolha o que melhor se encaixa para você
    </PlansSubtitle>

    <PlansContent>
      {PLANS.map((plan, i) => (
        <Plan key={plan.id} plan={plan} i={i} />
      ))}
    </PlansContent>

    <Box
      sx={{
        px: { xs: 2, md: 10 },
        mt: 1,
        mb: 2,
      }}
    >
      <PlansDescriptionText sx={{ my: 2 }}>
        <b>Gratuito:</b> Recomendado para iniciantes, é uma ótima forma de
        testar a plataforma. Você pode testar a plataforma. Você pode criar um
        anúncio gratuíto da sua propriedade, com as informações gerais do
        imóvel, e dura 7 dias.
      </PlansDescriptionText>

      <PlansDescriptionText sx={{ my: 2 }}>
        <b>Premium:</b> Recomendado para quem já tem conteúdos da propriedade
        como fotos, detalhes do imóvel como tipo de solo, atividades,
        topografia. Ele custa R$ 50,00 e seu anúncio ganha mais visibilidade
        dentro da plataforma.
      </PlansDescriptionText>

      <PlansDescriptionText sx={{ my: 2 }}>
        <b>Profissional:</b> Entre em contato conosco para conversarmos e
        conhecermos melhor o seu imóvel rural. Iremos até a propriedade,
        colheremos as informações mais importanjtes dela e faremos excelentes
        gravações explorando o imóvel através de dones de ponta. Essa é a
        maneira mais fácil de vender.
      </PlansDescriptionText>
    </Box>
  </Box>
);
