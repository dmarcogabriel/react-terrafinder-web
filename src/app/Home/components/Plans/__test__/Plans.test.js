import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
// import { render } from '@testing-library/react';
import Plans from '..';

describe('<Plans />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Plans />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  // Add tests here...

  it('matches snapshot', () => {
    const tree = renderer.create(<Plans />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
