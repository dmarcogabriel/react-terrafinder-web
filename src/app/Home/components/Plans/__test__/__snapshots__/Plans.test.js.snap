// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Plans /> matches snapshot 1`] = `
<div
  className="MuiBox-root css-uwkh73"
>
  <h4
    className="MuiTypography-root MuiTypography-h4 css-10ultvr-MuiTypography-root"
  >
    Conheça nossos planos de anúncio
  </h4>
  <h5
    className="MuiTypography-root MuiTypography-h5 css-i7xlex-MuiTypography-root"
  >
    Escolha o que melhor se encaixa para você
  </h5>
  <div
    className="MuiBox-root css-hqsvoc"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-9ww2aq-MuiPaper-root-MuiCard-root"
    >
      <div
        className="MuiBox-root css-1q82e8"
      >
        <p
          className="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
        >
          Grátis
        </p>
        <p
          className="MuiTypography-root MuiTypography-body2 css-1py0gvv-MuiTypography-root"
        />
      </div>
      <div
        className="MuiCardContent-root css-6z18u9-MuiCardContent-root"
      >
        <p
          className="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
        >
          Básico
        </p>
        <ul
          className="MuiList-root MuiList-padding css-1vsvtj8-MuiList-root"
        >
          <li
            className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
            disabled={false}
          >
            <div
              className="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1w3q91v-MuiSvgIcon-root"
                data-testid="CheckIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M9 16.17 4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                />
              </svg>
            </div>
            <div
              className="MuiListItemText-root css-tlelie-MuiListItemText-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
              >
                Duração de 7 dias
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
            disabled={false}
          >
            <div
              className="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1w3q91v-MuiSvgIcon-root"
                data-testid="CheckIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M9 16.17 4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                />
              </svg>
            </div>
            <div
              className="MuiListItemText-root css-tlelie-MuiListItemText-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
              >
                Até 15 fotos
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
            disabled={false}
          >
            <div
              className="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1w3q91v-MuiSvgIcon-root"
                data-testid="CloseIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                />
              </svg>
            </div>
            <div
              className="MuiListItemText-root css-tlelie-MuiListItemText-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
              >
                Sem demarcação do imóvel no mapa
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
            disabled={false}
          >
            <div
              className="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1w3q91v-MuiSvgIcon-root"
                data-testid="CloseIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                />
              </svg>
            </div>
            <div
              className="MuiListItemText-root css-tlelie-MuiListItemText-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
              >
                Sem suporte
              </p>
            </div>
          </li>
        </ul>
        <button
          className="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root css-3xtg2n-MuiButtonBase-root-MuiButton-root"
          data-testid="plan-button-p1"
          disabled={false}
          onBlur={[Function]}
          onContextMenu={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          Quero esse
        </button>
      </div>
    </div>
    <div
      className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-1vtrsxw-MuiPaper-root-MuiCard-root"
    >
      <div
        className="MuiBox-root css-1enlqw"
      >
        <p
          className="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
        >
          R$ 50
        </p>
        <p
          className="MuiTypography-root MuiTypography-body2 css-1py0gvv-MuiTypography-root"
        >
          por anúncio
        </p>
      </div>
      <div
        className="MuiCardContent-root css-trkami-MuiCardContent-root"
      >
        <p
          className="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
        >
          PREMIUM
        </p>
        <ul
          className="MuiList-root MuiList-padding css-1vsvtj8-MuiList-root"
        >
          <li
            className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
            disabled={false}
          >
            <div
              className="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1w3q91v-MuiSvgIcon-root"
                data-testid="CheckIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M9 16.17 4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                />
              </svg>
            </div>
            <div
              className="MuiListItemText-root css-tlelie-MuiListItemText-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
              >
                Duração do anúncio de 90 dias
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
            disabled={false}
          >
            <div
              className="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1w3q91v-MuiSvgIcon-root"
                data-testid="CheckIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M9 16.17 4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                />
              </svg>
            </div>
            <div
              className="MuiListItemText-root css-tlelie-MuiListItemText-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
              >
                Visibilidade aumentada na plataforma
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
            disabled={false}
          >
            <div
              className="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1w3q91v-MuiSvgIcon-root"
                data-testid="CheckIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M9 16.17 4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                />
              </svg>
            </div>
            <div
              className="MuiListItemText-root css-tlelie-MuiListItemText-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
              >
                Até 30 fotos e 1 vídeo
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
            disabled={false}
          >
            <div
              className="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1w3q91v-MuiSvgIcon-root"
                data-testid="CloseIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                />
              </svg>
            </div>
            <div
              className="MuiListItemText-root css-tlelie-MuiListItemText-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
              >
                Sem filmagens por drone nem tratamento de fotos
              </p>
            </div>
          </li>
        </ul>
        <button
          className="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root css-z4v7o8-MuiButtonBase-root-MuiButton-root"
          data-testid="plan-button-p2"
          disabled={false}
          onBlur={[Function]}
          onContextMenu={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          Quero esse
        </button>
      </div>
    </div>
    <div
      className="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiCard-root css-9ww2aq-MuiPaper-root-MuiCard-root"
    >
      <div
        className="MuiBox-root css-1q5hivw"
      >
        <p
          className="MuiTypography-root MuiTypography-h3 css-gepadz-MuiTypography-root"
        >
          A definir
        </p>
        <p
          className="MuiTypography-root MuiTypography-body2 css-1py0gvv-MuiTypography-root"
        >
          Faça uma cotação com nossos especialistas
        </p>
      </div>
      <div
        className="MuiCardContent-root css-fuewyq-MuiCardContent-root"
      >
        <p
          className="MuiTypography-root MuiTypography-h5 css-ag7rrr-MuiTypography-root"
        >
          Profissional
        </p>
        <ul
          className="MuiList-root MuiList-padding css-1vsvtj8-MuiList-root"
        >
          <li
            className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
            disabled={false}
          >
            <div
              className="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1w3q91v-MuiSvgIcon-root"
                data-testid="CheckIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M9 16.17 4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                />
              </svg>
            </div>
            <div
              className="MuiListItemText-root css-tlelie-MuiListItemText-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
              >
                Duração de até 365 dias
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
            disabled={false}
          >
            <div
              className="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1w3q91v-MuiSvgIcon-root"
                data-testid="CheckIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M9 16.17 4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                />
              </svg>
            </div>
            <div
              className="MuiListItemText-root css-tlelie-MuiListItemText-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
              >
                Maior visibilidade da plataforma
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
            disabled={false}
          >
            <div
              className="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1w3q91v-MuiSvgIcon-root"
                data-testid="CheckIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M9 16.17 4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                />
              </svg>
            </div>
            <div
              className="MuiListItemText-root css-tlelie-MuiListItemText-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
              >
                Gravação da propriedade com drones e tratamento de imagens
              </p>
            </div>
          </li>
          <li
            className="MuiListItem-root MuiListItem-gutters css-1nxmd3h-MuiListItem-root"
            disabled={false}
          >
            <div
              className="MuiListItemIcon-root css-cveggr-MuiListItemIcon-root"
            >
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1w3q91v-MuiSvgIcon-root"
                data-testid="CheckIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M9 16.17 4.83 12l-1.42 1.41L9 19 21 7l-1.41-1.41z"
                />
              </svg>
            </div>
            <div
              className="MuiListItemText-root css-tlelie-MuiListItemText-root"
            >
              <p
                className="MuiTypography-root MuiTypography-body2 css-e784if-MuiTypography-root"
              >
                Suporte exclusivo
              </p>
            </div>
          </li>
        </ul>
        <button
          className="MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium MuiButtonBase-root css-1t7ftwz-MuiButtonBase-root-MuiButton-root"
          data-testid="plan-button-p3"
          disabled={false}
          onBlur={[Function]}
          onContextMenu={[Function]}
          onDragLeave={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          Quero esse
        </button>
      </div>
    </div>
  </div>
  <div
    className="MuiBox-root css-1s8uw1l"
  >
    <p
      className="MuiTypography-root MuiTypography-body1 css-1fyq76y-MuiTypography-root"
    >
      <b>
        Gratuito:
      </b>
       Recomendado para iniciantes, é uma ótima forma de testar a plataforma. Você pode testar a plataforma. Você pode criar um anúncio gratuíto da sua propriedade, com as informações gerais do imóvel, e dura 7 dias.
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1 css-1fyq76y-MuiTypography-root"
    >
      <b>
        Premium:
      </b>
       Recomendado para quem já tem conteúdos da propriedade como fotos, detalhes do imóvel como tipo de solo, atividades, topografia. Ele custa R$ 50,00 e seu anúncio ganha mais visibilidade dentro da plataforma.
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1 css-1fyq76y-MuiTypography-root"
    >
      <b>
        Profissional:
      </b>
       Entre em contato conosco para conversarmos e conhecermos melhor o seu imóvel rural. Iremos até a propriedade, colheremos as informações mais importanjtes dela e faremos excelentes gravações explorando o imóvel através de dones de ponta. Essa é a maneira mais fácil de vender.
    </p>
  </div>
</div>
`;
